app/forms/address_attributes.rb

```diff
@@ -11,6 +11,8 @@ module AddressAttributes
     validates :address_post_code, post_code: true, length: { maximum: POSTCODE_LENGTH }
   end
 
+  # rubocop:disable MethodLength
```

Comment:
@leoapost I tried but there was a problem. I will revisit this when I have more time if that's ok with you?